libs/ui-components/src/components/Fleet/CreateFleet/steps/UpdatePolicyStep.tsx

```diff
@@ -0,0 +1,247 @@
+import WithHelperText from '../../../common/WithHelperText';
+
+export const updatePolicyStepId = 'update-policy';
+
+export const isUpdatePolicyStepValid = (errors: FormikErrors<FleetFormValues>) => !errors.rolloutPolicy;
+
+const RolloutPolicyBatch = ({ index }: { index: number }) => {
+  const { t } = useTranslation();
+
+  const items = React.useMemo(
+    () => ({
+      percent: t('Percentage'),
+      value: t('Number of devices'),
+    }),
+    [t],
+  );
+
+  const [{ value: batch }, meta] = useField<BatchForm>(`rolloutPolicy.batches.${index}`);
+
+  const isPercent = batch.limitType === BatchLimitType.BatchLimitPercent;
+  return (
+    <ExpandableFormSection
+      title={t('Batch {{ batchNum }}', { batchNum: index + 1 })}
+      fieldName={`rolloutPolicy.batches.${index}`}
+    >
+      <Grid hasGutter>
+        {/* Show errors not related to an individual field */}
+        {typeof meta.error === 'string' && (
+          <GridItem>
+            <ErrorHelperText meta={meta} />
+          </GridItem>
+        )}
+        <FormGroup label={t('Select devices using labels')}>
+          <LabelsField aria-label={t('Label selector')} name={`rolloutPolicy.batches.${index}.selector`} />
+        </FormGroup>
+        <FormGroup label={t('Select a subset using')}>
+          <Flex flexWrap={{ default: 'nowrap' }} alignItems={{ default: 'alignItemsFlexEnd' }}>
+            <FlexItem>
+              <FormSelect items={items} name={`rolloutPolicy.batches.${index}.limitType`} />
+            </FlexItem>
+            <FlexItem>
+              <NumberField
+                aria-label={t('Numeric selector')}
+                name={`rolloutPolicy.batches.${index}.limit`}
+                min={1}
+                max={isPercent ? 100 : undefined}
+                widthChars={isPercent ? 3 : 8}
+              />
+            </FlexItem>
+            {isPercent && <FlexItem>%</FlexItem>}
+          </Flex>
+        </FormGroup>
+        <FormGroup
+          isRequired
+          label={
+            <WithHelperText
+              ariaLabel={t('Success threshold')}
+              content={t(
+                'Minimum percentage of devices that must update successfully in order to continue updating the next batch of devices.',
+              )}
+              showLabel
+            />
+          }
+        >
+          <Flex flexWrap={{ default: 'wrap' }}>
+            <FlexItem>{t('If')} </FlexItem>
+            <FlexItem>
+              <NumberField
+                aria-label={t('Success threshold')}
+                name={`rolloutPolicy.batches.${index}.successThreshold`}
+                min={1}
+                max={100}
+                isRequired
+              />
+            </FlexItem>
+            <FlexItem flex={{ lg: 'flex_1' }} style={{ minWidth: 200 }}>
+              {t("% of the batch devices pass the success criteria, move to next batch or the rest of fleet's devices")}
+              .
```

Comment:
I guess `dot` should be in `t`
```suggestion
              {t("% of the batch devices pass the success criteria, move to next batch or the rest of fleet's devices.")}
```

Reply:
Good eye, modified.